server/memcached/src/main/java/org/infinispan/server/memcached/MemcachedMetadata.java

```diff
@@ -34,6 +34,11 @@ private MemcachedMetadata(int flags, long lifespan, EntryVersion version) {
       this.flags = flags;
    }
 
+   @Override
+   public boolean isEmpty() {
+      return super.isEmpty() && flags != 0;
```

Comment:
typo? 
```suggestion
      return super.isEmpty() && flags == 0;
```

Reply:
Yes, good catch. This is from before when I had the code as the inverse (isPresent or w/e I called it). Fixing.